app/test/unit/git/branch-test.ts

```diff
@@ -72,6 +72,7 @@ describe('git/branch', () => {
       expect(onBranch.branch.tip.sha).toEqual(
         'dfa96676b65e1c0ed43ca25492252a5e384c8efd'
       )
+      expect(onBranch.branch.tip.shortSha).toEqual('dfa9667')
```

Comment:
[MRW](https://imgur.com/gallery/zSRy2pG/comment/576095257) I see all the tests being updating to look at this new field:

![](https://user-images.githubusercontent.com/359239/51476223-3f230580-1d5b-11e9-8143-9b7e8a5fafdb.gif)